samples/ComponentExamples/README.md

```diff
@@ -0,0 +1,20 @@
+# UI Library Component and Binding Examples 
+
+This folder contains several mini examples of ACS UI Library component and binding.
+
+## When should I use component instead of composite
+
+If you haven't checked our composite, we strongly recommend you take a look at our composite first. Composites are all-in-one react components which provides you a full ACS experience within several lines of code. 
```

Comment:
It might be better to just have one line referring to composite, and then link to storybook directly. (avoid duplicating reasoning etc)

Reply:
I've checked the storybook, looks like we don't put these reasons in storybook yet, I will keep it here for now, but open a work item to add these reasons into storybook (then remove them here)